acra-censor/acra-censor_implementation.go

```diff
@@ -72,6 +75,13 @@ func (acraCensor *AcraCensor) HandleQuery(rawQuery string) error {
 	normalizedQuery, queryWithHiddenValues, parsedQuery, err := common.HandleRawSQLQuery(rawQuery)
 	if err == common.ErrQuerySyntaxError {
 		acraCensor.logger.WithError(err).Warning("Failed to parse input query")
+		if acraCensor.parseErrorsLogPath != "" {
+			err := acraCensor.saveQuery(rawQuery)
+			if err != nil {
+				acraCensor.logger.WithError(err).Errorf("An error occurred while saving unparsable query")
+				return err
```

Comment:
I think if censor configured with `acracensor.ignoreParseError` then we should not stop processing on error related with unparsed query and enough log error and try to save without stopping processing.

Reply:
@vixentael what you think about this?

Reply:
agree, if `acracensor.ignoreParseError` is true, then log error and move forward